cmd/headers/download/downloader.go

```diff
@@ -56,7 +56,11 @@ func GrpcSentryClient(ctx context.Context, sentryAddr string) (proto_sentry.Sent
 	return proto_sentry.NewSentryClient(conn), nil
 }
 
-func RecvUploadMessage(ctx context.Context, sentry proto_sentry.SentryClient, handleInboundMessage func(ctx context.Context, inreq *proto_sentry.InboundMessage, sentry proto_sentry.SentryClient) error) {
+func RecvUploadMessage(ctx context.Context,
```

Comment:
Please add comment explaining about `wg` used only for tests like in the `RecvMessage`